app/controllers/Promotion.scala

```diff
@@ -20,20 +20,21 @@ import views.support.{BillingPeriod => _}
 
 object Promotion extends Controller with LazyLogging with CatalogProvider {
 
-  def getAdjustedRatePlans(promo: AnyPromotion, country:Country)(implicit tpBackend:TouchpointBackend): Option[Map[String, String]] = {
+  def getAdjustedRatePlans(promo: AnyPromotion, country:Country, requestedCurrency: Option[Currency])(implicit tpBackend:TouchpointBackend): Option[Map[String, String]] = {
+    def impliedCurrency = CountryGroup.byCountryCode(country.alpha2).getOrElse(CountryGroup.UK).currency
```

Comment:
Logged in Trello: https://trello.com/c/coarykrm/108-tidy-up-code-in-checkout-scala-for-trackingcode